doc/how_to_guides/parallel_planning/config/parallel_planning_moveit_cpp.yaml

```
@@ -0,0 +1,53 @@
+# Default configurations. More information can be found in the moveit_cpp and planning scene monitor tutorials
+planning_scene_monitor_options:
+  name: "planning_scene_monitor"
+  robot_description: "robot_description"
+  joint_state_topic: "/joint_states"
+  attached_collision_object_topic: "/moveit_cpp/planning_scene_monitor"
+  publish_planning_scene_topic: "/moveit_cpp/publish_planning_scene"
+  monitored_planning_scene_topic: "/moveit_cpp/monitored_planning_scene"
+  wait_for_initial_state_timeout: 10.0
+
+# Define which planning pipelines are loaded by moveit_cpp. These loaded pipelines will be available for simple and parallel planning
+planning_pipelines:
+  pipeline_names: ["ompl", "pilz_industrial_motion_planner", "chomp", "ompl_rrts"]
```

Might be a bug here:

```suggestion
  pipeline_names: ["ompl_rrtc", "pilz_industrial_motion_planner", "chomp", "ompl_rrts"]
```

Also, I would be in favor of more verbose names like `ompl_rrt_connect` and `ompl_rrt_star`. Could also leave the `ompl_` out if you prefer.